modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -90,22 +88,16 @@ InputParameters validParams<FeatureFloodCount>()
   params.addRequiredCoupledVar("variable", "The variable(s) for which to find connected regions of interests, i.e. \"features\".");
   params.addParam<Real>("threshold", 0.5, "The threshold value for which a new feature may be started");
   params.addParam<Real>("connecting_threshold", "The threshold for which an existing feature may be extended (defaults to \"threshold\")");
-  params.addParam<Real>("volume_threshold", "The threshold used for calculating feature volumes (defaults to \"threshold\")");
   params.addParam<bool>("use_single_map", true, "Determine whether information is tracked per coupled variable or consolidated into one (default: true)");
   params.addParam<bool>("condense_map_info", false, "Determines whether we condense all the node values when in multimap mode (default: false)");
   params.addParam<bool>("use_global_numbering", true, "Determine whether or not global numbers are used to label features on multiple maps (default: true)");
-  params.addParam<bool>("enable_var_coloring", false, "Instruct the UO to populate the variable index map.");
-  params.addParam<bool>("compute_halo_maps", false, "Instruct the UO to communicate proper halo information to all ranks");
+  params.addParam<bool>("enable_var_coloring", false, "Instruct the Postprocessor to populate the variable index map.");
+  params.addParam<bool>("compute_halo_maps", false, "Instruct the Postprocessor to communicate proper halo information to all ranks");
```

Comment:
Seriously, there's actually a lot of extra work and communication to make those maps and they are almost never used and probably won't be in a real simulation. If you want to make an image of the halos, you have to turn that variable on. I think I'd rather leave it in to avoid the extra expense and time.